tests/php/test_class-wc-connect-shipping-method.php

```diff
@@ -103,31 +103,6 @@ public function test_is_valid_package_destination( $package, $expected ) {
 	/**
 	 * @covers WC_Connect_Shipping_Method::calculate_shipping
 	 */
-	public function test_invalid_destination_calculate_shipping() {
```

Comment:
I think @DanReyLop is correct - we could however use a test for a not-yet-setup service schema - to make sure we handle that correctly. @DanReyLop - would you write up an issue to add a test_not_yet_setup_shipping_method test that ensures log is called if a service settings mock get_service_settings call returns an empty array

With that, I think we can move forward with this pr